finetune/base.py

```diff
@@ -508,36 +522,36 @@ def generate_text(self, seed_text="", max_length=None, use_extra_toks=None):
         """
         if use_extra_toks is None:
             use_extra_toks = self._trained
-    
+            
```

Comment:
we probably want to get rid of all of this extra whitespace (other examples as well)